Python/Product/PythonTools/PythonTools.csproj

```diff
@@ -271,9 +271,6 @@
     <Compile Include="..\Analysis\IEnumerableExtensions.cs">
       <Link>IEnumerableExtensions.cs</Link>
     </Compile>
-    <Compile Include="..\Analysis\Interpreter\PythonInterpreterFactoryExtensions.cs">

```

Comment:
It's public in Analysis.dll, so rather than compiling it into multiple DLLs and causing conflicts, I made the one function that was needed public.